src/sentry/api/endpoints/relay/project_configs.py

```diff
@@ -43,6 +52,11 @@ def _post(self, request: Request):
         relay = request.relay
         assert relay is not None  # should be provided during Authentication
 
+        if request.relay_request_data.get("globalConfig"):
+            metrics.incr("relay.project_configs.global.fetched")
+            global_config = get_global_config()
+            return Response(global_config, status=200)
```

Comment:
nit: This works, but it would be just as easy to add the global config to the response in `_post_by_key` and allow the client to request global and project-specific configs at once.

This might not be a use case in Relay, but it is good practice to not make any server-side assumptions about what the client-side wants.

Reply:
Also note that the [Relay side](https://github.com/getsentry/relay/pull/2320/files#diff-9e66add7db47c3f18146ebef0427032761cc7c185870ff3e603e73441e585b2fR83) expects `{"global": {...}}`.

Reply:
hmm then I would re-introduce the ?version parameter i suppose?

Reply:
> then I would re-introduce the ?version parameter i suppose?

That parameter already exists, and I don't think we'd need any changes to it in this iteration.

My suggestion is to change the last line of `_post_by_key` from

```python
return Response({"configs": configs}, status=200)
```

to 

```python
response = {"configs": configs}
if request.relay_request_data.get("globalConfig"):
    metrics.incr("relay.project_configs.global.fetched")
    response["global"] = get_global_config()
    
return Response(response, status=200)
```

Reply:
so my point was that there's three different functions used to get the project configs based on the version, and now with global config we won't even necessarily include a project config version when we make the request, so thats why it didnt seem fitting to me to put it inside those functions. 
 
I made a refactor to make it possible to request both project configs and global configs though